### ISSUE TITLE
Door Sensor Showing even if not existent

### ISSUE THREAD

**Issue by BNoiZe** [ebaauw/homebridge-nb#29]
I just recently noticed, that the plugin is also showing a door sensor. I still have nuki version 1 without a sensor. Ist there some way to hide it manually?

The documentation states "The when present and configured, door sensor is exposed as separate Contact Sensor service, including Eve history.". Still it is showing up for me.

**Comment by ebaauw** [comment]
What’s the output of `nb list`?

**Comment by BNoiZe** [comment]
```
[
  {
    "deviceType": 0,
    "firmwareVersion": "1.12.0",
    "lastKnownState": {
      "batteryChargeState": 62,
      "batteryCharging": false,
      "batteryCritical": false,
      "mode": 2,
      "state": 1,
      "stateName": "locked",
      "timestamp": "2021-03-15T05:00:11+00:00"
    },
    "name": "Zuhause",
    "nukiId": 265575041
  },
  {
    "deviceType": 2,
    "firmwareVersion": "1.6.4",
    "lastKnownState": {
      "batteryCritical": false,
      "mode": 2,
      "ringactionState": false,
      "ringactionTimestamp": "2021-03-13T10:25:59+00:00",
      "state": 1,
      "stateName": "online",
      "timestamp": "2021-03-15T04:40:54+00:00"
    },
    "name": "Opener",
    "nukiId": 509253866
  }
]
```

**Comment by ebaauw** [comment]
Code checks whether `lastKnowState.doorSensorState` equals 0 (deactivated), but not whether it it present.

**Comment by BNoiZe** [comment]
It would also be possible to check against `firmwareVersion`. The firmware for Nuki Smart Lock v1 without door sensor is 1.x.x. Version 2 is numbered 2.x.x.

**Comment by ebaauw** [comment]
Could you try beta v1.1.1-0?

It checks against `null` (present) or `0` (deactivated).  It should delete the cached accessory when the door sensor isn't present or activated, and create/restore it otherwise.

**Comment by BNoiZe** [comment]
beta v1.1.1-0 does fix it. The door sensor is gone now.

**Comment by BNoiZe** [comment]
But homebridge-nb does throw an error now. Looks like it is trying to remove the accessory over and over again.

`[4/3/2021, 12:37:02 PM] [Nuki] Nuki_Bridge_B9FC91: warning: heartbeat error: Cannot find the bridged Accessory to remove.`

**Comment by ebaauw** [comment]
That's not good.  It'll probably stop trying to after you've restarted Homebridge.  Beta v1.1.1-1 should fix it.

**Comment by BNoiZe** [comment]
Fixed with the latest beta.

**Comment by ebaauw** [comment]
Published v1.1.1.
